### PR TITLE
read q connection string from keyvault

### PR DESCRIPTION
**Pull request by jasonpaige**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -8,6 +8,19 @@ def component = "consumer"
 
 def channel = '#rpe-build-notices'
 
+List<LinkedHashMap<String, Object>> secrets = [
+  secret('send-letter-queue-listen-connection-string', 'SERVICE_BUS_CONNECTION_STRING')
+]
+
+static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
```
Comment (nitinprabhu) [inline Jenkinsfile_CNP:15]: I am not sure but if using something like `LinkedHashMap<String, Object>` will throw Serializable exception as Object is not Serializable?

Comment (nitinprabhu) [reply Jenkinsfile_CNP:15]: I had a look at CMC code and its using the same so looks like it works :)

File: infrastructure/output.tf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+output "vaultUri" {
+  value = "https://send-letter-${var.env}.vault.azure.net/"
+}
+
+output "vaultName" {
+  value = "send-letter-queue-listen-connection-string"
+}
```
Comment (rkondratowicz) [inline infrastructure/output.tf:7]: where are those variables used? Can we add a comment on it in this file?

**Comment by rkondratowicz** [comment]
pr is 20 days old. do we want this change merged?

**Comment by lgonczar** [comment]
Should this be closed now?

**Pull request closed without merging**